CAVEATS.md

```diff
@@ -0,0 +1,8 @@
+# Caveats
```

Comment:
I'm not sure it is worth putting this in separate file.

Reply:
Yeah I'm merging the README part into #278. The caveats file/contents is too early to be merged.